src/isomorphic/classic/class/ReactClass.js

```diff
@@ -518,13 +529,30 @@ function mixSpecIntoComponent(Constructor, spec) {
           } else if (specPolicy === SpecPolicy.DEFINE_MANY) {
             proto[name] = createChainedFunction(proto[name], property);
           }
+
+          if (__DEV__) {
+            if (isFromMixin) {
```

Comment:
These 5 lines seem identical in three different places. Can we move this `__DEV__` block outside the conditions and write this once?